src/zui/ZUICreatePerson/InfoInputForm.tsx

```diff
@@ -0,0 +1,64 @@
+import globalMessageIds from 'core/i18n/globalMessageIds';
+import messageIds from 'zui/l10n/messageIds';
+import { TextField } from '@mui/material';
+import { FC, MutableRefObject } from 'react';
+import { Msg, useMessages } from 'core/i18n';
+import {
+  ZetkinCreatePerson,
+  ZetkinPersonNativeFields,
+} from 'utils/types/zetkin';
+
+interface InfoInputFormProps {
+  field: keyof ZetkinCreatePerson;
+  label?: string;
+  onChange: (field: string, value: string) => void;
+  required?: boolean;
+  isURLField?: boolean;
+  inputRef?: MutableRefObject<HTMLInputElement | undefined> | undefined;
+  style?: Record<string, unknown>;
+  error?: boolean;
+}
+const InfoInputForm: FC<InfoInputFormProps> = ({
+  field,
+  label,
+  onChange,
+  required,
+  style,
+  inputRef,
+  isURLField,
+  error,
+}) => {
+  const globalMessages = useMessages(globalMessageIds);
+
+  return (
+    <TextField
+      error={error}
+      fullWidth
+      helperText={
+        error && (
```

Comment:
I think we should add another check here, so that the field has been blurred (lost focus). Because right now, the error will display _while_ I'm typing, which is confusing. I think it should only show up once I blur the feld.